src/node_contextify.cc

```diff
@@ -1611,7 +1611,8 @@ static std::vector<std::string_view> throws_only_in_cjs_error_messages = {
     "Identifier '__filename' has already been declared",
     "Identifier '__dirname' has already been declared",
     "await is only valid in async functions and "
-    "the top level bodies of modules"};
+    "the top level bodies of modules",
+};
```

Comment:
We avoid unnecessary changes to keep the git history as simple as possible.

```suggestion
    "the top level bodies of modules"};
```